enhancements/network/routed-ingress-primary-udn-static-ips.md

```diff
@@ -0,0 +1,723 @@
+   Motivation sections. These should be easy and explain why the community
+   should desire this enhancement.
+1. **Create a PR.** Assign it to folks with expertise in that domain to help
+   sponsor the process.
+1. **Merge after reaching consensus.** Merge when there is consensus
+   that the design is complete and all reviewer questions have been
+   answered so that work can begin.  Come back and update the document
+   if important details (API field names, workflow, etc.) change
+   during code review.
+1. **Keep all required headers.** If a section does not apply to an
+   enhancement, explain why but do not remove the section. This part
+   of the process is enforced by the linter CI job.
+
+See ../README.md for background behind these instructions.
+
+Start by filling out the header with the metadata for this enhancement.
+
+# Routed ingress support for primary UDN attached VMs with static IPs
+
+## Summary
+
+There's an ongoing interest to migrate Virtual Machines (VMs) from other
+virtualization platforms into OpenShift. These users are both after a managed IP
+experience (which they have on their existing platform), and also want to
+consume existing Kubernetes features, like network policies, and sometimes
+services.
+
+Traditional virtualization users have some expectations on what a
+virtualization platform should provide. Live-migration, and IP address
+persistence across reboots are paramount features for any virtualization
+solution, and OpenShift Virtualization currently supports these features on its
+primary UserDefinedNetworks (UDNs).
+
+These users have additional requirements, like routed ingress into their VMs,
+and, to import the VMs with the MAC, IPs, and gateway configuration the VM had
+on the previous platform.
+
+Routed ingress into the VMs is being added to OpenShift already, and is tracked
+by [this enhancement](https://github.com/openshift/enhancements/pull/1636/files)
+which tracks adding BGP support to OpenShift.
+
+Given the aggressive time-frame we're attempting, adding support for importing
+a VM into OpenShift with it's MAC, IPs, and gateway on top of primary UDN is the
+missing piece we should pursue.
+
+## Motivation
+
+Some users are running VMs in virtualization platforms having a managed IP
+configuration.
+
+For this kind of users, we need to have a way to enable their existing VMs to
+run properly after being migrated into OpenShift, without any guest
+configuration changes. For that, we need to import the VMs from their existing
+platforms, preserving their existing MACs, IPs, and gateway configuration.
+
+### User Stories
+
+- As the owner of a VM running in a traditional virtualization platform, I want
+to import said VM - whose IPs were statically configured - into Kubernetes,
+attaching it to an overlay network. I want to ingress/egress using the same IP
+address (the one assgined to the VM).
+- As the owner of a VM running in a traditional virtualization platform, I want
```

Comment:
Is this localnet ipamful ?

Reply:
No - "As the owner of a VM running in a traditional virtualization platform, I want to import said VM into Kubernetes, attaching it to an **overlay network**"

Reply:
But is said 
```
I want to
have managed IPs for my VMs
```

Reply:
Yes, but I still don't see the implication.

If it helps, this is about importing a VM into a layer2 overlay with IPAM.